defaults/main.yml

```diff
@@ -1,5 +1,18 @@
 ---
 apache_mirror: http://apache.mirrors.tds.net
-kafka_id: 0
+kafka_hosts: "{{ansible_fqdn}}:9092"
 kafka_version: 0.8.1.1
 skip_install: False
+
+kafka_num_network_threads: 2
+kafka_num_io_threads: 2
+kafka_num_partitions: 2
+kafka_socket_send_buffer_bytes: 1048576
+kafka_socket_receive_buffer_bytes: 1048576
+kafka_socket_request_max_bytes: 104857600
+kafka_log_flush_interval_messages: 10000
+kafka_log_flush_interval_ms: 1000
+kafka_log_retention_hours: 24
+kafka_log_segment_bytes: 536870912
+kafka_log_cleanup_interval_mins: 1
+zookeeper_connection_timeout_ms: 1000000
```

Comment:
This was contributed in https://github.com/hpcloud-mon/ansible-kafka/pull/5 presumably because some tweaking of these settings was desired.